src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -7413,6 +7413,68 @@ def test_aks_azure_service_mesh_with_ingress_gateway(self, resource_group, resou
             self.is_empty(),
         ])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_azure_service_mesh_with_egress_gateway(self, resource_group, resource_group_location):
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=79813556&view=results) to validate the change.

Reply:
test passed